examples/bloom_filter/arrow_bf_host_bulk_example.cu

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
```

Comment:
This file is the exact replica of the `bloom_filter_host_bulk_example.cu` except it uses the `arrow_bf_policy`. Was thinking of removing this and incorporating both of these policies in the same example by throwing contents in a function templated over policy.